rules-reviewed/fuse/sonicesb/sonic-esb.windup.xml

```diff
@@ -94,23 +91,22 @@
             </when>
             <perform>
                  <hint title="Reference to com.sonicsw.xq.XQInitContext" effort="4" severity="mandatory">
-                     <message><![CDATA[
-
-                             The XQInitContext is used to receive initialization information for the service from Sonic ESB.
-
-                             This is not neccessary for Camel.  The init() method that receives this object should be replaced with Spring Bean property injection.  For initialization beyond propery injection, leverage Spring's @PostConstruct annotation on this init() method.
-
-                             * [Spring @PostConstruct Documentation](http://docs.spring.io/spring/docs/2.5.x/reference/beans.html#beans-postconstruct-and-predestroy-annotations)
-
-                             ```java
-                             @PostConstruct
-                             public void init() \{
-                                 //leverage injected properties
-                             }
-                             ```
-
-
-                     ]]></message>
+                     <message>
+                       <![CDATA[
+                       The XQInitContext is used to receive initialization information for the service from Sonic ESB.
+
+                       This is not neccessary for Camel.  The init() method that receives this object should be replaced with Spring Bean property injection.  For initialization beyond propery injection, leverage Spring's @PostConstruct annotation on this init() method.
+
+                       * [Spring @PostConstruct Documentation](http://docs.spring.io/spring/docs/2.5.x/reference/beans.html#beans-postconstruct-and-predestroy-annotations)
+
+                       ```java
+                       @PostConstruct
+                       public void init() \{
```

Comment:
Not related to this commit, but:
I think that we should revisit how the Rewrite construct {...} is matched.
Forcing users to escape every '{' doesn't seem like a good UX to me.